### PR TITLE
Add Dark LAF, move to AWT FileDialog

### PR DESCRIPTION
**Pull request by modmuss50**
**Issues to resolve before merge:**

I could not find a awt Directory browser, anyone have any suggestions? Or just leave it? @unascribed might be able to help suggest something?

### CODE REVIEW

**Comment by unascribed** [comment]
I don't believe AWT supports a directory browser. Forgot about that.

Since LWJGL3 is modular, you *could* just pull in the nfd module and use that.

**Comment by modmuss50** [comment]
Ok, there is it being done with lwgl, seems a bit extreme but it works.

I also need to stop jenkins building PR's as its uploading them to maven :/

File: build.gradle [hunk 1]
```diff
@@ -77,7 +78,16 @@ dependencies {
     application 'de.sciss:syntaxpane:1.1.+'
 	application name: "darcula", version: "1.0.0"
 
-    testCompile 'junit:junit:4.+'
+	application 'org.lwjgl:lwjgl:3.1.2'
```
Comment (unascribed) [inline build.gradle]: Wrong indentation

File: build.gradle [hunk 2]
```diff
@@ -63,16 +68,25 @@ configurations {
...
 dependencies {
     compile 'com.google.guava:guava:21.+'
     compile 'org.javassist:javassist:3.21.0-GA'
     compile 'org.bitbucket.mstrobel:procyon-compilertools:0.5.33.8-enigma'
     compile 'com.google.code.gson:gson:2.8.1'
 
-
     application 'de.sciss:syntaxpane:1.1.+'
+	application name: "darcula", version: "1.0.0"
+
```
Comment (unascribed) [inline build.gradle]: Minor; wrong indentation.

**Comment by modmuss50** [comment]
I think there a few places where the formatting is wrong, I can just auto format the whole repo once this PR is merged.

**Comment by modmuss50** [comment]
@Thog did you have any input on this?

File: build.gradle [hunk 3]
```diff
@@ -63,16 +68,25 @@ configurations {
...
+	application 'org.lwjgl.osgi:org.lwjgl.nfd:3.1.2'
+
+	application "org.lwjgl:lwjgl:3.1.2:natives-windows"
+	application "org.lwjgl:lwjgl:3.1.2:natives-linux"
+	application "org.lwjgl:lwjgl:3.1.2:natives-macos"
 
-    application 'de.sciss:syntaxpane:1.1.+'
 
-    testCompile 'junit:junit:4.+'
+	testCompile 'junit:junit:4.+'
```
Comment (Thog) [inline build.gradle]: This must be spaces

**Comment by Thog** [comment]
Also, do you have checked that the natives of lwjgl are correctly loaded when launched as a jar?

**Comment by modmuss50** [comment]
Ok, I can do that. I think LWJGL natives should be loaded from the jar, I will also test on nix just to make sure

**Comment by modmuss50** [comment]
screen shot 2017-06-28 at 15 59 38

Seems to work just fine when being ran from the jar on windows and mac os

**Comment by modmuss50** [comment]
Wait, hold on

**Comment by modmuss50** [comment]
screen shot 2017-06-28 at 16 38 30

Uhh, i forgot its only the directory browsers that are using lwjgl. and they dont work. I will have to look into it.

**Comment by asiekierka** [comment]
"Merged".

**Pull request closed without merging**